File: tier1/java/91ac4bd5/Handler.java
public final class Handler {
    public static String handle(String name) {
        StringBuilder encoded = new StringBuilder();
        for (char c : name.toCharArray()) {
            switch (c) {
                case '<': encoded.append("&lt;"); break;
                case '>': encoded.append("&gt;"); break;
                case '&': encoded.append("&amp;"); break;
                case '"': encoded.append("&quot;"); break;
                case '\'': encoded.append("&#x27;"); break;
                default: encoded.append(c);
            }
        }
        return "<div>" + encoded + "</div>";
    }
}

File: tier1/javascript/fae0d38e/handler.js
function handle(name) {
  const encoded = String(name).replace(/[&<>"'/]/g,
    (c) => ({ '&': '&amp;', '<': '&lt;', '>': '&gt;',
              '"': '&quot;', "'": '&#x27;', '/': '&#x2F;' })[c]);
  return '<div title="' + encoded + '">report</div>';
}

module.exports = { handle };
